zwave_js_server/const.py

```diff
@@ -3,12 +3,14 @@
 from typing import Dict, List
 
 # minimal server schema version we can handle
-MIN_SERVER_SCHEMA_VERSION = 3
+MIN_SERVER_SCHEMA_VERSION = 4
```

Comment:
Do we need to release a new version of the server before we merge here?

Reply:
We need to release a new version of the server before we **release**. We can hold off on merge until we do that, but every PR thats open right now, plus one I haven't submitted yet to add commands related to statistics collection, is dependent on that release

Reply:
I'll let you coordinate the releases and merges. :+1: